### PR TITLE
No dots in include

### PR DESCRIPTION
**Pull request by Christian-B**
Got rid of all or most of the #include ".. UGLINESS!

### CODE REVIEW

File: neural_modelling/src/neuron/builds/neural_build.mk [hunk 1]
```diff
@@ -131,12 +135,15 @@ $(foreach obj, $(STDP), $(eval $(call stdp_rule, $(obj))))
 $(WEIGHT_DEPENDENCE_O): $(WEIGHT_DEPENDENCE) $(SYNAPSE_TYPE_H)
 	-mkdir -p $(dir $@)
 	$(CC) -D__FILE__=\"$(notdir $*.c)\" -DLOG_LEVEL=$(PLASTIC_DEBUG) $(CFLAGS) \
+	        $(INCLUDE_PLASTICITY_HEADERS) \
 	        -include $(SYNAPSE_TYPE_H) -o $@ $<
 
 $(TIMING_DEPENDENCE_O): $(TIMING_DEPENDENCE) $(SYNAPSE_TYPE_H) \
                         $(WEIGHT_DEPENDENCE_H)
+	echo TIMING_DEPENDENCE_O
```
Comment (dkfellows) [inline neural_modelling/src/neuron/builds/neural_build.mk]: ????

File: neural_modelling/src/delay_extension/Makefile [hunk 2]
```diff
@@ -3,3 +3,5 @@ BUILD_DIR = build/
 SOURCES = delay_extension.c
 
 include ../common.mk
+
+CFLAGS += -I $(NEURAL_MODELLING_DIRS)/src/common
```
Comment (dkfellows) [inline neural_modelling/src/delay_extension/Makefile]: I believe that there shouldn't usually be a space between `-I` and the directory

File: neural_modelling/src/neuron/c_main.c [hunk 3]
```diff
@@ -15,13 +15,13 @@
  *
  */
 
-#include "../common/in_spikes.h"
+#include <in_spikes.h>
 #include "neuron.h"
```
Comment (dkfellows) [inline neural_modelling/src/neuron/c_main.c]: Surely we should be consistent here and use `#include <…>` for all of these?

File: neural_modelling/src/neuron/builds/neural_build.mk (hunk 1 above)
Comment (Christian-B) [reply neural_modelling/src/neuron/builds/neural_build.mk]: debug gone

File: neural_modelling/src/delay_extension/Makefile (hunk 2 above)
Comment (Christian-B) [reply neural_modelling/src/delay_extension/Makefile]: Our pattern is to include the space

See:  lines 54 and 55 in
https://github.com/SpiNNakerManchester/spinnaker_tools/blob/master/make/Makefile.common

The space may not be required but that is out of scope for this PR

File: neural_modelling/src/neuron/c_main.c (hunk 3 above)
Comment (Christian-B) [reply neural_modelling/src/neuron/c_main.c]: The difference is that <in_spikes.h> is brought in by an -I ... while "neuron.h" is local to c_main.c

May be possible/ cleaner to do all as <...> but that is also out of scope of this PR

**Pull request merged**